tests/test_pg_catalog_query.py

```
@@ -0,0 +1,133 @@
+    import pyarrow as pa
+
+    def send_batch(batch: pa.RecordBatch, callback):
+        rdr = pa.RecordBatchReader.from_batches(batch.schema, [batch])
+        if hasattr(rdr, "__arrow_c_stream__"):
+            capsule = rdr.__arrow_c_stream__()
+        else:
+            from pyarrow.cffi import export_stream  # type: ignore
+            capsule = export_stream(rdr)
+        callback(capsule)
+
+    def handle_query(sql, callback, **kwargs):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)